docs/user/network/dashmate/index.rst

```diff
@@ -494,6 +513,166 @@ To disable logging to a file outside the container, reset the log path to ``null
+         "path": "/home/ubuntu/logs/gateway.log",
+         "template": null
+      },      
+      {
+         "type": "stdout",
+         "format": "text",
+         "template": null
+      }
+   ]'
+
+**Disable file logging**
+
+To disable logging to a file, remove the file config from the accessLogs setting:
+
+.. code-block:: shell
+
+   dashmate config set platform.gateway.log.accessLogs '[
+      {
+         "type": "stdout",
+         "format": "text",
+         "template": null
+      }
+   ]'
+
+**View log settings**
+
+To view the current Platform gateway log settings, run:
+
+.. code-block:: shell
+
+   dashmate config get platform.gateway.log
+
+.. _dashmate-logs-platform-abci:
+
+Drive ABCI logs
+^^^^^^^^^^^^^^^
+
+**Enable file logging**
+
+Use ``dashmate config set`` to configure a location for storing Platform ABCI logs on the host
+file system. Replace the example path with one that matches your system:
+
+.. code-block:: shell
+
+   dashmate config set platform.drive.abci.logs '{
+      "stdout": {
+         "destination":"stdout",
+         "level": "info",
+         "format":"compact",
+         "color":true
+      }, 
+      "file": {
+         "destination": "/home/ubuntu/logs/drive-abci.log",
+         "level": "info",
+         "format": "compact",
+         "color": true
+      }
+   }'
+
+**Disable file logging**
+
+To disable logging to a file, remove the file config from the logs setting:
+
+.. code-block:: shell
+
+   dashmate config set platform.drive.abci.logs '{
+      "stdout": {
+         "destination":"stdout",
+         "level": "info",
+         "format":"compact",
+         "color":true
+      }
+   }'
+
+**View log settings**
+
+To view the current Platform gateway log settings, run:
+
+.. code-block:: shell
+
+   dashmate config get platform.drive.abci.logs
+
+.. _dashmate-logs-platform-tenderdash:
+
+Tenderdash logs
+^^^^^^^^^^^^^^^
+
+**Enable file logging**
+
+Use ``dashmate config set`` to configure a location for storing Tenderdash logs on the host file
+system. Replace the example path with one that matches your system:
+
+.. code-block:: shell
+
+   dashmate config set platform.drive.tenderdash.log.path "/home/ubuntu/logs/drive-tenderdash.log"
+
+**Disable file logging**
+
+To disable logging to a file, set the path back to ``null``:
+
+.. code-block:: shell
+
+   dashmate config set platform.drive.tenderdash.log.path null
+
+**View log settings**
+
+To view the current Tenderdash log settings, run:
+
+.. code-block:: shell
+
+   dashmate config get platform.drive.tenderdash.log
+
+.. _dashmate-logs-change-level:
+
+Change log level
+^^^^^^^^^^^^^^^^
+
+Platform services support standard log levels. In increasing order of detail they are: ``error``,
+``warn``, ``info``, ``debug``, and `trace`. By default, logging is set to the ``info`` level. The
+log level for Platform services can be changed by using ``dashmate config set`` to update the
+following values:
+
+* platform.gateway.log.level
```

Comment:
Access logs have logging level too.

Reply:
I don't see a level setting specifically for [access logs in the JSON](https://github.com/dashpay/platform/blob/v1.4-dev/packages/dashmate/src/config/configJsonSchema.js#L673-L787). Does `platform.gateway.log.level` set the level for both service logs and access logs?